SystemInformer/mainwnd.c

```diff
@@ -284,46 +284,161 @@ PPH_STRING PhMwpInitializeWindowTitle(
+        else if (!(ElevationType == TokenElevationTypeFull || ElevationType == TokenElevationTypeDefault) && !isRunningAsSYSTEMUser && isHighILOrGreater)
+            PhAppendStringBuilder2(&stringBuilder, L" (Elevation : IL). ");                                                                                 // Token = Full (UAC)
+        else if ((ElevationType == TokenElevationTypeFull || ElevationType == TokenElevationTypeDefault) && (isRunningAsSYSTEMUser || isHighILOrGreater))
+            PhAppendStringBuilder2(&stringBuilder, L" (Elevated). ");
+
+
+        // 5. Driver connection verbosity
+        //
+        KPH_LEVEL driverLevel = KphLevelEx(FALSE);
+        BOOL isConnectedToDriver = driverLevel != KphLevelNone;
+        if (!isConnectedToDriver)
+            PhAppendStringBuilder2(&stringBuilder, L"  |  <No-Driver> ");
+        else
+            PhAppendStringBuilder2(&stringBuilder, L"  |   ");
+
+        // 5.5 - Driver Load type (Features + Security)
+        //
+        switch (driverLevel)
+        {
+        case KphLevelMax:
+            PhAppendStringBuilder2(&stringBuilder, L"Driver Connection: ++[Maximum]  ");
+            break;
+        case KphLevelHigh:
+            PhAppendStringBuilder2(&stringBuilder, L"Driver Connection: +[High]  ");
+            break;
+        case KphLevelMed:
+            PhAppendStringBuilder2(&stringBuilder, L"Driver Connection: ~[Medium]  ");
+            break;
+        case KphLevelLow:
+            PhAppendStringBuilder2(&stringBuilder, L"Driver Connection: -[Lower]  ");
+            break;
+        case KphLevelMin:
+            PhAppendStringBuilder2(&stringBuilder, L"Driver Connection: --[Minimal]  ");
+            break;
+        }
+
+        // 6. Session ID \ WindowStation \ Desktop
+        //
+        ULONG sessionID;
+        PhGetProcessSessionId(NtCurrentProcess(), &sessionID);
+        PPH_STRING winStaOfProcess = PhGetCurrentWindowStationName();          // Always WinSta0 for anything interactive
+        PPH_STRING desktopNameOfThread = PhGetCurrentThreadDesktopName();
+        PhAppendFormatStringBuilder(&stringBuilder, L" |    %lu \\ ", sessionID);
+        PhAppendStringBuilder(&stringBuilder, &winStaOfProcess->sr);
+        PhAppendStringBuilder2(&stringBuilder, L" \\ ");
+        PhAppendStringBuilder(&stringBuilder, &desktopNameOfThread->sr);
+        PhAppendStringBuilder2(&stringBuilder, L" ");
+        PhDereferenceObject(winStaOfProcess);
+        PhDereferenceObject(desktopNameOfThread);
+
+        // 7. A dev is running this under the Visual Stduio Debugger?
+        //
+        if (IsDebuggerPresent())
+            PhAppendStringBuilder2(&stringBuilder, L"           ** BEING DEBUGGED **");
```

Comment:
`PhIsDebuggerPresent`, also might be useful to show information `PhGetKernelDebuggerInformation` too?

Also comment is a little misleading, I suggest removing "under the Visual Studio".

There is also a lot of white space. I'm not sure that's helpful if you have a narrower window. I get that it is meant to be "verbose" but I think we're not using the title space most effectively.